src/data/faq.json

```diff
@@ -30,16 +30,86 @@
                 "icon": "/assets/img/icons/info_faq.svg",
                 "sections": [
                     {
-                        "title": "General Questions",
+                        "title": "General questions",
                         "id": "ramp_down_general",
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "General topics about the end of development of the Corona-Warn-App will soon be published here.",
+                                "title": "What is the background for the end of the development of the CWA?",
                                 "anchor": "ramp_down",
                                 "active": false,
                                 "textblock": [
-                                    ""
+                                    "Considering increased population immunity, easily transmissible viral variants, and a return to public life (mostly) without Corona measures, a contact tracing app is currently no longer needed. Beginning June 1, 2023, the CWA will be placed in a hibernation mode. Until April 30, 2023, it will be possible to warn other users through the CWA regarding an increased risk of infection.",

```

Comment:
The privacy document refers to this as "sleep" mode, not "hibernation". See https://github.com/corona-warn-app/cwa-app-android/blob/release/3.2.x/Corona-Warn-App/src/main/assets/privacy_en.html

It would be good to use the same term everywhere, i.e. choose either "sleep" or "hibernation".

Whether you use "sleep" or "hibernation" you would also need to explain what could wake up the app from its "sleep" state.

"sleep" => with an analogy to human sleep would be the next day
"hibernation" => with an analogy to animal winter hibernation would be waking up after the winter

Reply:
Latest german phrase here is: "Ruhe-Modus", which points more to the direction of "hibernation".
And we are going to redirect any further question regarding this phase (including your expected explanation for a "wake up" procedure) to the client, sorry. Technically, there is no sufficient answer to this question.